recipes/caf/all/conanfile.py

```diff
@@ -0,0 +1,140 @@
+import os
+from conans import ConanFile, CMake, tools
+from conans.errors import ConanInvalidConfiguration
+from conans.tools import Version
+
+
+class CAFConan(ConanFile):
+    name = "caf"
+    description = "An open source implementation of the Actor Model in C++"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/actor-framework/actor-framework"
+    topics = "conan", "caf", "actor-framework", "actor-model", "pattern-matching", "actors"
+    license = "BSD-3-Clause, BSL-1.0"
+    exports_sources = ["CMakeLists.txt", "patches/**"]
+    generators = "cmake", "cmake_find_package"
+    settings = "os", "compiler", "build_type", "arch"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+        "log_level": ["ERROR", "WARNING", "INFO", "DEBUG", "TRACE", "QUIET"],
+        "openssl": [True, False]
+    }
+    default_options = {"shared": False, "fPIC": True, "log_level": "QUIET", "openssl": True}
+    _source_subfolder = "source_subfolder"
+    _build_subfolder = "build_subfolder"
+    _cmake = None
+
+    @property
+    def _is_static(self):
+        return 'shared' not in self.options.values.keys() or not self.options.shared
```

Comment:
Don't need it, use `not self.options.shared` instead
```suggestion
```

Reply:
Right - this isn't needed anymore.